otx/algorithms/classification/tasks/inference.py

```diff
@@ -435,7 +436,11 @@ def patch_color_conversion(pipeline):
             for pipeline_step in cfg.pipeline:
                 if subset == "train" and pipeline_step.type == "Collect":
                     pipeline_step = get_meta_keys(pipeline_step)
-            patch_color_conversion(cfg.pipeline)
+
+            if isinstance(cfg.pipeline, ConfigDict):
+                cfg.pipeline = {key: patch_color_conversion(val) for key, val in cfg.pipeline.items()}
```

Comment:
if you want patch_color_conversion() to all values in cfg.pipeline, then how about to use code as below?
`for val in cfg.pipeline.values(): patch_color_conversion(val)`
and remove `return` in patch_color_conversion().

Reply:
Removed these lines by using BranchImage. Thank you for suggestion.